### ISSUE TITLE
Problems with Python 3.8 and higher as well as loading LAS package in anaconda

### ISSUE THREAD

**Issue by eucalypt1** [kinverarity1/lasio#493]
Anaconda 2.03 and Python 3.8.8 64 bit.
I am not able to load any lasio library (including vs 0.29). 
![Lasio compat problem](https://user-images.githubusercontent.com/82836107/137203512-2e235b18-e70e-4644-aaba-1d5d5fd33280.jpg)

I tried with and without using a version number and pip install lasio.  The latter states that lasio is allready there but it doesn't show up in the base environment (including when updating the index). Just doesn't show.  

Basically I am stuck with our software development which uses lasio a fair bit and is not running, VS is failing at the import statements.  Any suggestions
